cmd/mountlib/mount.go

```diff
@@ -22,6 +25,30 @@ var (
 	ExtraFlags         *[]string
 )
 
+// Check is folder is empty
+func checkMountEmpty(mountpoint string) error {
+	fp, fpErr := os.Open(mountpoint)
+	defer fs.CheckClose(fp, &fpErr)
+
+	if fpErr != nil {
+		return errors.Wrap(fpErr, "Can not open: " + mountpoint)
+	}
+
+	_, fpErr = fp.Readdirnames(1)
+
+	// directory is not empty
+	if fpErr != io.EOF {
+		var e error
+		if fpErr == nil {
+			e = errors.New("Directory is not empty: " + mountpoint)
+		} else {
+			e = errors.Wrap(fpErr, "Directory is not empty: " + mountpoint)
```

Comment:
This line has styling error:
`-			e = errors.Wrap(fpErr, "Directory is not empty: " + mountpoint)`
`+			e = errors.Wrap(fpErr, "Directory is not empty: "+mountpoint)`

but I think it is false positive error.

Reply:
Just run the code through [gofmt](https://golang.org/cmd/gofmt/) - that is the go way!  Or even better get [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) to run on save in your editor.

Reply:
@ncw 
So should I correct it or leave it as is.
Personally I think code looks better as is right now. ( I`m new to go and I`m basing my opinion on coding style roles in other languages).

Reply:
Correct it please, all code in rclone has been run through gofmt - you'll find the same in nearly all go projects.  There is only one correct way to format go programs - whatever gofmt outputs!